featurebyte/session/snowflake.py

```diff
@@ -104,7 +104,7 @@ async def list_databases(self) -> list[str]:
         -------
         list[str]
         """
-        databases = await self.execute_query("SHOW DATABASES")
+        databases = await self.execute_query("SHOW DATABASES", timeout=INTERACTIVE_SESSION_TIMEOUT)
```

Comment:
Would it make sense to have a method `execute_query_interactive` that applies this timeout by default, similar to `execute_query_long_running`?

Reply:
That makes sense